Gemfile

```diff
@@ -3,6 +3,8 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby "3.1.2"
 
+gem "pry-byebug"
```

Comment:
you might move `pry` to development and testing!

Reply:
noted